package.json

```diff
@@ -91,6 +91,59 @@
             }
           }
         }
+      },
+      {
+        "name": "apply_projectTemplate",
```

Comment:
bleh, don't like this name. what could be better? 

"generate_Sample"? 
"create_ExampleProject"?
"make_SampleProject"?

Reply:
maybe... I _think_ you can open a new template in your current workspace though

Reply:
I'd set `"canBeReferencedInPrompt": true` here, and either go with `generateProject` as the name or stick with the `<verb>_<noun>` pattern and set `"toolReferenceName": "generateProject"` as what will be shown in the reference (e.g. `@Confluent help me #generateProject in golang`)